homework/style.css

```diff
@@ -1,3 +1,137 @@
+html {
```

Comment:
You still need to work on responsiveness.

![image](https://user-images.githubusercontent.com/2788771/61959209-3558b400-afc3-11e9-89d1-cc648cae5838.png)